### PR TITLE
Starting with HTML file #78

### PR DESCRIPTION
**Pull request by clburns107**

### CODE REVIEW

**Comment by clburns107** [comment]
how do i comment out code in scss?

/\* code */  doesnt seem to turn it grey like it does in a css file.

**Comment by halfghaninne** [comment]
The [Sass documentation](http://sass-lang.com/documentation/file.SCSS_FOR_SASS_USERS.html) shows this as an example:

```
/* This comment is
 * several lines long.
 * since it uses the CSS comment syntax,
 * it will appear in the CSS output. */
body { color: black; }

// These comments are only one line long each.
// They won't appear in the CSS output,
// since they use the single-line comment syntax.
a { color: green; }
```

**Comment by clburns107** [comment]
Ok im trying to merge my HTML back into the view for the user dashboard. I dont want to merge my branch yet i just want to see if it will work on my computer first.  

When i moved my code from the static page and integrated it into the users/home page i got an error that i needed to migrate.  When i run bundle exec rake db:migrate iterm gives this error:

Could not find octokit-4.3.0 in any of the sources
Run `bundle install` to install missing gems.

When i run bundle install iterm then gives this error:
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
An error occurred while installing rugged (0.24.0), and Bundler cannot continue.
Make sure that `gem install rugged -v '0.24.0'` succeeds before bundling.

**Comment by sumeetjain** [comment]
Did you see the message yesterday about running `brew install cmake`? Give that a go and then try your other commands again.

File: app/views/static_pages/users_dashboard.html.erb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <div class="col-12 title-box">
+      <h2>Show Attendance 2016</h2>
+    </div>
+  </div>
+
+  <div class="clear content">
+    <div class="col-12 content-box">
+      <div class="clear">
+        <div class="col-6">
+
```
Comment (sumeetjain) [inline app/views/static_pages/users_dashboard.html.erb:38]: Remove trailing whitespace

**Pull request merged**